src/debug/daccess/dacimpl.h

```diff
@@ -3904,7 +3904,8 @@ class EnumMethodInstances
 #define DAC_ENTER() \
     EnterCriticalSection(&g_dacCritSec); \
     ClrDataAccess* __prevDacImpl = g_dacImpl; \
-    g_dacImpl = this;
+    g_dacImpl = this; \
+    g_gcDacGlobals = &g_gc_dac_vars;
```

Comment:
>g_gcDacGlobals = &g_gc_dac_vars; [](start = 4, length = 32)

I think you need to remove this line now that g_gcDacGlobals is the published DAC global already.